src/plots/cartesian/axes.js

```diff
@@ -1682,7 +1682,7 @@ axes.doTicksSingle = function(gd, arg, skipTitle) {
     var valsClipped = vals.filter(clipEnds);
 
     // don't clip angular values
-    if(ax._id === 'angular') {
+    if(ax._id === 'angularaxis') {
```

Comment:
not new in this PR, but I think you have an `axid` here.

Reply:
went one step further in 0c517d197dce032bcd5835adf80952672c2341fe